### PR TITLE
Text Styles 

### PR DESCRIPTION
**Pull request by jennminerva**
Styled text for default.html and blog.html. Blog Index looks like the h1 size is quite large, but that may be the design, so I didn't mess with it, as I wasn't asked to.

Also, I changed the margins on .BlogContainer img so that they wouldn't be offset. I put it in a separate commit in case that was intentional and you wanted to undo it.

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
 			<div class="line"></div>
 			<p class="underlined BlogBack"><a href="{{ site.url }}">Back to Home</a></p>
 		</div>
-		<div class="column col-xs-12 col-md-8 BlogContainer">
+		<div class="postContent column col-xs-12 col-md-8 BlogContainer">
```
Comment (jeremypbeasley) [inline _layouts/default.html]: is .BlogContainer still relevant? If not, feel free to remove it. Also, as a general rule add specific classes after generic ones. You don't want the styles you write for this to be overridden by `.column` etc.

File: _resources/_styles/base.styl [hunk 2]
```diff
@@ -93,10 +93,10 @@ img
             margin-top -2vw
     img
         width 100vw
-        margin 2em 0 2em -11vw
+        margin 2em 0
```
Comment (jeremypbeasley) [inline _resources/_styles/base.styl]: Consider moving this declaration out of `.BlogContainer` and under `.postContent`. Also, can you please write all new CSS in valid CSS format, not the punctuation-less stylus format. We're eventually going to need to clean it up but why not avoid adding more to clean up ahead of time.

File: _resources/_styles/type.styl [hunk 3]
```diff
@@ -140,19 +140,39 @@ h3, h4
...
+.postContent h3 {
+	font-size: 1rem;
+}
+.postContent em {
+	font-family: typeface-h3;
+	margin: 2em 0;
+	display: block;
+}
+
+sup {
```
Comment (jeremypbeasley) [inline _resources/_styles/type.styl]: I don't see where you can ever make a <sup> with markdown. If that's the case, let's remove this.

File: _resources/_styles/type.styl [hunk 4]
```diff
@@ -140,19 +140,39 @@ h3, h4
...
+	font-size 50%;
+	top -1em;
+}
+ol {
```
Comment (jeremypbeasley) [inline _resources/_styles/type.styl]: Why are we making ordered lists semi-opaque?

File: package.json [hunk 5]
```diff
@@ -15,7 +15,6 @@
     "jquery": "latest",
     "normalize.css": "latest",
     "flexboxgrid": "5.0.0",
-    "jquery": "latest",
```
Comment (jeremypbeasley) [inline package.json:18]: nice catch!

File: _resources/_styles/type.styl [hunk 6]
```diff
@@ -140,19 +140,39 @@ h3, h4
...
+}
+.postContent h2 {
+	font-size: 1.7rem;
+}
+@media laptop {
+	.postContent h2 {
+		font-size: 2rem;
+	}
+}
+.postContent h3 {
```
Comment (jeremypbeasley) [inline _resources/_styles/type.styl]: Let's add an H4 style as well. I know in older posts, there's a few of ####

File: _assets/master.css [hunk 7]
```diff
@@ -1775,23 +1775,37 @@ h4 {
...
+    font-size: 2rem;
+  }
+}
+.postContent h3 {
+  font-size: 1rem;
 }
-.post strong,
-.post em {
+.postContent em {
   font-family: CircularStdBook, CircularStdBold, sans-serif;
```
Comment (jeremypbeasley) [inline _assets/master.css]: Why this font stack? First bold then... fall back to bold? Please use the variables in vars.styl if you can.

File: _resources/_styles/type.styl [hunk 8]
```diff
@@ -140,19 +140,39 @@ h3, h4
...
-		margin 2em 0
-		display block
-	sup
-		font-size 50%
-		top -1em
-	ol
-		opacity .5
+// POSTS
+	
+.postContent h1 {
```
Comment (jeremypbeasley) [inline _resources/_styles/type.styl]: Consistency in code is a lot more important than perfection. I'm using thing kind of  CamelCase on all classes on this project so let's make it `PostContent`

**Review by jeremypbeasley — CHANGES_REQUESTED** [review]

**Comment by jeremypbeasley** [comment]
One of the things to consider is if on the `default` layout, we even forced the `title` into a spot or leave it to the author to determine if it's appropriate to begin with a big H1 title or something else. 

This also gets us out of having to downscale `.postContent` styles from other styles. In other words, an H1 in a post is the same size and treatment as an an H1 on, say, the home page.

**Comment by jennminerva** [comment]
@jeremypbeasley, pretty sure I got everything fixed in the footer. 

Removing the `<h2>` from the title of default markdown files made it look wonky and broken, so I restored it. Let me know if you'd rather me do something else.

**Review by jeremypbeasley — APPROVED** [review]

**Comment by jeremypbeasley** [comment]
I had to do a pretty major refactor here and due to conflicts, ported over to another PR here: https://github.com/jeremypbeasley/realescape/pull/32

**Pull request closed without merging**